about.html

```diff
@@ -15,6 +15,8 @@
     />
     <link rel="stylesheet" href="./public/css/main.css" />
     <link rel="stylesheet" href="./public/css/about.css" />
+    
+    <!--Shujun: I think the bootstrap script part should be put before the end of the body part-->
```

Comment:
You are correct! It is indeed best practice to put scripts to the end so that they are loaded last and it speeds up page load. Does not matter too much in this case as the script is small enough.

Apparently other solution is to use <script async> or <script defer> tags, just FYI.